bwc-test/src/test/java/SecurityBackwardsCompatibilityIT.java

```diff
@@ -73,6 +135,12 @@ public void testBasicBackwardsCompatibility() throws Exception {
         }
     }
 
+    @SuppressWarnings("unchecked")
+    public void testWhoAmI() throws Exception {
```

Comment:
Simple WhoAmI check to ensure the plugin is installed and enabled. If disabled this would fail with an error indicating that the WhoAmI endpoint was not registered.

Reply:
Thanks, I downloaded the BWC test output and the test ran we can build up the test more as well go.
![image](https://github.com/opensearch-project/security/assets/2754967/c1414b43-3980-407e-9eac-44decf4c6171)